src/app/users/users-update/users-update.component.ts

```diff
@@ -177,4 +179,10 @@ export class UsersUpdateComponent implements OnInit {
     });
   }
 
+  ngAfterViewInit(): void {
+    this.imageUploader.hasPreview = true;
+    this.imageUploader.imgSrc = 'assets/image.png';
```

Comment:
@ScottHuangNYU `this.imageUploader.imgSrc` should be assigned to the user's profile image src (if they have one)